src/noos_inv/docker.py

```diff
@@ -52,13 +53,16 @@ def _dockerhub_login(ctx: Context, user: str, token: Optional[str]) -> None:
 
 
 @task
-def build(ctx, name=None, context=None, arg=None):
+def build(ctx, name=None, file=None, context=None, arg=None):
     """Build Docker image locally."""
     name = name or ctx.docker.name
+    file = file or ctx.docker.file
```

Comment:
Maybe,

```python
file = file or f"{ctx.docker.context}/{ctx.docker.file}"
```

Reply:
isn't docker default PATH/Dockerfile and not dockercontext/Dockerfile ?